torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -1872,6 +1872,11 @@ def generate_py_arg_inner(lines, raw_arg, arg_type):
                     raise NotImplementedError(
                         f"arg type {arg_type} with raw_arg {raw_arg}, {type(raw_arg)} is not yet supported by custom_op_wrapper"
                     )
+            elif isinstance(raw_arg, torch.device):
+                # device
+                self.include_extra_header("torch/csrc/Device.h")
```

Comment:
In general, we should try to avoid directly including torch headers for compilation speed reason, but I see we are already doing that for dtype and layout. We may need to revisit this at some point.

Reply:
@desertfire Understood. Should I open an issue to put that in our backlog?

Reply:
Feel free to do that, since we already talked about reducing compilation time as our next goal.